data/default_release_images.json

```diff
@@ -32,17 +32,17 @@
         "cpu_architectures": [
             "x86_64"
         ],
-        "url": "quay.io/openshift-release-dev/ocp-release:4.11.46-x86_64",
-        "version": "4.11.46"
+        "url": "quay.io/openshift-release-dev/ocp-release:4.11.47-x86_64",
```

Comment:
Why do you need to update that in this PR? It's better that this change will be changed as part of separate PR if it's not related to the problem that this PR fixs

Reply:
It's the same as [master](https://github.com/openshift/assisted-service/blob/master/data/default_release_images.json#L35)

Reply:
fixed